catkin_home/src/object_detector/src/detection2d.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env python3
+# USAGE
+# python3 detection2d.py
+
+import numpy as np
+import argparse
+import tensorflow as tf
+import cv2
+import pathlib
+import rospy
+import threading
+import imutils
+import time
+from imutils.video import FPS
+from sensor_msgs.msg import CompressedImage
+from geometry_msgs.msg import Pose
+from std_msgs.msg import Bool
+from object_detector.msg import objectDetection, objectDetectionArray
+from object_detection.utils import label_map_util
+from object_detection.utils import visualization_utils as vis_util
+
+SOURCES = {
+    "VIDEO": str(pathlib.Path(__file__).parent) + "/../resources/test.mp4",
+    "CAMERA": 0,
+    "ROS_IMG": "/camaras/0",
```

Comment:
`"/camaras/0"` is constant?

Reply:
The idea is that it can be overwritten by SOURCE parameter as it can be seen in object_detection launch file.
![image](https://user-images.githubusercontent.com/36351179/162880157-be2c0075-35de-4f47-b9b7-510db2499c72.png)